src/jobs/push-to-app-catalog.yaml

```
@@ -24,7 +24,8 @@ steps:
   - prepare-catalogbot-git-ssh
   - helm-chart-template:
       chart: <<parameters.chart>>
-  - helm-lint
+  - helm-lint:
+      charts: "helm/<< parameters.chart >>"
```

Let's just pass a single chart because the command has a **way** better output when single chart is passed.

```suggestion
      chart: "<< parameters.chart >>"
```

---

As above... Most cases like from this job, it will be a single chart so the output will be fine. Just not sure there is any value is restricting the command